### ISSUE TITLE
Accessibility: Options cannot be changed via TAB navigation, Show/Hide results does not highlight when focused (only changes on hover).

### ISSUE THREAD

**Issue by John-Paul-R** [John-Paul-R/PoE-Atlas-Website#56]

**Comment by John-Paul-R** [comment]
Fixed by 66b86421a0f19c9d20ba54288f5a63c75e86506c,
